cmd/libs/go2idl/parser/parse.go

```diff
@@ -316,6 +319,12 @@ func (b *Builder) priorCommentLines(pos token.Pos) string {
 	return ""
 }
 
+func tcFuncNameToName(in string) types.Name {
+	name := strings.TrimLeft(in, "func ")
+	nameParts := strings.Split(name, "(")
+	return tcNameToName(nameParts[0])
```

Comment:
Is it possible that we will get unnamed function here? Then nameParts[0] can be something strange...